src/scalper.ts

```diff
@@ -467,7 +467,8 @@ export class Scalper {
     
     // Sleep for the max time of the reruns then kill thread
     await sleepExact((1 + percentDrift) * scalperWindow);
-    return;
+    console.log(this.symbol, "Remove stale gamma fill listener", gammaBidID, gammaAskID)
+    fillFeed.removeEventListener('message', gammaFillListener);
```

Comment:
Still testing this to make sure it doesn't intercept and kill a gammaFillListener from a current thread. Not sure the scope here if you know

Reply:
![image](https://user-images.githubusercontent.com/107803907/192149813-c993cc46-d74c-407a-bdc6-bb3aa8c0f614.png)
![image](https://user-images.githubusercontent.com/107803907/192149822-3e8f7cf4-be78-4bba-9c02-a93a73bf4e1f.png)
correctly removed the old ones
![image](https://user-images.githubusercontent.com/107803907/192149838-c1a58ad1-10a3-40da-b033-eee1a8c09e85.png)